generator_process.py

```diff
@@ -161,6 +137,38 @@ def writeException(fatal, e):
         writeStr(e)
         stdout.flush()
 
+    try:
+        from absolute_path import absolute_path
+        # Support Apple Silicon GPUs as much as possible.
+        os.environ["PYTORCH_ENABLE_MPS_FALLBACK"] = "1"
+        sys.path.append(absolute_path("stable_diffusion/"))
+        sys.path.append(absolute_path("stable_diffusion/src/clip"))
+        sys.path.append(absolute_path("stable_diffusion/src/k-diffusion"))
+        sys.path.append(absolute_path("stable_diffusion/src/taming-transformers"))
+
+        site.addsitedir(absolute_path(".python_dependencies"))
+        import pkg_resources
+        pkg_resources._initialize_master_working_set()
+
+        from stable_diffusion.ldm.generate import Generate
+        from omegaconf import OmegaConf
+        from PIL import ImageOps
+        from io import StringIO
+    except Exception as e:
```

Comment:
```suggestion
    except ModuleNotFoundError:
```
This would be more resilient than string checking if it works.

Reply:
Yes it works. I'll still want it to catch any other exception just in case though.